SPECS-EXTENDED/python-xmltodict/python-xmltodict.spec

```diff
@@ -1,18 +1,16 @@
 %global pypi_name xmltodict
 
 Name:               python-xmltodict
-Version:            0.12.0
-Release:            15%{?dist}
+Version:            0.13.0
+Release:            6%{?dist}
 Summary:            Python to transform XML to JSON
-Vendor:		    Microsoft Corporation
-Distribution:   Azure Linux
+
 License:            MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
 URL:                https://github.com/martinblech/xmltodict
-Source0:            %{url}/archive/v%{version}/%{pypi_name}-%{version}.tar.gz
+Source0:            %{url}/archive/v%{version}/%{pypi_name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
I don't think this change is needed. One of the PR checks is failing around the same. Can you please take a look.